ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -137,8 +137,8 @@ BOOTSTRAP_NODE_TYPE=${arch_instance_type}.large
 workers=${COMPUTE_NODE_REPLICAS:-3}
 if [[ "${COMPUTE_NODE_REPLICAS}" -le 0 ]]; then
     workers=0
-elif [[ "${COMPUTE_NODE_REPLICAS}" -gt 5 ]]; then
-    workers=5
+elif [[ "${COMPUTE_NODE_REPLICAS}" -gt 9 ]]; then
```

Comment:
Instead of changing this at cluster creation time, you could scale the nodes after they get created

Reply:
> Instead of changing this at cluster creation time, you could scale the nodes after they get created

Sure I will keep this in mind.
Any reason why we should not create those nodes during install time?

Reply:
I just think we shouldnt change code that effects everyone if we dont have to

Reply:
I hear you, I am not altering the default value which still remains at 3
this is a harmless change - removing the cap, as a user can now create more `COMPUTE_NODES` nodes during install time.

Reply:
This was removed in this recent PR 
https://github.com/openshift/release/pull/41737